src/wix/WixToolset.Core.WindowsInstaller/Unbind/ExtractCabinetsCommand.cs

```diff
@@ -55,13 +55,21 @@ public void Execute()
             // index all of the cabinet files
             if (OutputType.Module == this.Output.Type || this.TreatOutputAsModule)
             {
+                var mediaRow = new MediaRow(null, WindowsInstallerTableDefinitions.Media);
+                mediaRow[0] = 1;
+                mediaRow[1] = 1;
+                mediaRow[3] = "MergeModule.CABinet";
+
+                embeddedCabinetRowsByDiskId.Add(0, mediaRow);
```

Comment:
Should the `0` in the `Add()` first param be a `1` to match the `DiskId` of the `MergeModule.CABinet` Media?